File: Services/Models/Service.cs
using System.ComponentModel.DataAnnotations;

namespace Services.Models
{
	public class Service
	{
		[Key]
		[Required]
		public int Id { get; set; }

		[Required]
		public string Info { get; set; }

		[Required]
		public int UserId { get; set; }

		[Required]
		public int ServiceCategoryId { get; set; }

		// navigation properties
		public ServiceCategory ServiceCategory { get; set; }

		public User User { get; set; }

		public ICollection<Photo> Photos { get; set; }
	}
}

File: Client/src/app/_models/Service.ts
import { Photo } from "./Photo";
import { ServiceCategory } from "./ServiceCategory";
import { User } from "./User";

export class Service {
	constructor(public id: number, public info: string, public userId: number, public user: User, public serviceCategoryId: number, public serviceCategory: ServiceCategory, public photos: Photo[]) {
	}
}
